manage.py

```diff
@@ -0,0 +1,45 @@
+#!/usr/local/bin/python3
+"""Root module to manage flask script commands."""
+import os
+import unittest
+
+from flask_script import Command, Manager
+
+from mod_regression.controllers import mod_regression
+from mod_regression.update_regression import update_expected_results
+from run import app
+
+manager = Manager(app)
+
+
+@manager.add_command
+class UpdateResults(Command):
+    """
+    Update results for the present samples with new ccextractor version.
+
+    Pass path to CCExtractor binary as the first argument. Example, `python manage.py update /path/to/ccextractor`
+    """
+
+    name = 'update'
+    capture_all_args = True
+
+    def run(self, remaining):
+        """Driver function for update subcommand."""
+        if len(remaining) == 0:
+            print('path to ccextractor is missing')
+            return 1
+        else:
```

Comment:
This else is unneeded as you return in the if. Removes a level of indent and makes it a bit easier on the remaining structure too